src/System.Web.OData/packages.config

```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.AspNet.WebApi.Core" version="5.2.2" targetFramework="net45" />
   <package id="Microsoft.Extensions.DependencyInjection" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Extensions.DependencyInjection.Abstractions" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.OData.Core" version="7.2.0" targetFramework="net45" />
+  <!--<package id="Microsoft.OData.Core" version="7.2.0" targetFramework="net45" />
```

Comment:
Is this change intentional?

Reply:
I've rolled that back. However, this PR relies on changes in odata.net. Can I use nightly build for 7.4 here?

---
In reply to: [137667056](https://github.com/OData/WebApi/pull/1004#discussion_r137667056) [](ancestors = 137667056)